### PR TITLE
The automatic page refresh is not working in Wazuh Dashboard in version 4.9.0

### PR DESCRIPTION
**Pull request by guidomodarelli**
### Description
The automatic page refresh is not working properly for the Wazuh Dashboard in version 4.9.0. The page is refreshing but it is not updating the changes on the Dashboard. 
image
 
### Issues Resolved
[#25617](https://github.com/wazuh/wazuh/issues/25617)

### Evidence

https://github.com/user-attachments/assets/9246fc29-d12c-421c-8b20-b210f6b08d96

### Test
1. Remove the sample data, if present.
2. Add the sample data to obtain the latest timestamps.
3. Navigate to any application.
4. Set a refresh interval of 5 seconds within the time filter.
5. Observe whether the histogram updates over time.

> [!IMPORTANT]
> 1. Check if the URL updates with the savedQuery when you select an option or clear all (savedQueries)
> 2. Additionally, verify that the AppFilters are being saved correctly in the URL

https://github.com/user-attachments/assets/768486d4-2e5e-42e5-8436-d278b1fcdeb8

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [x] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by chantal-kelm — DISMISSED** [review]
CR: ✅ 
TEST: ✅ 


https://github.com/user-attachments/assets/79ec5426-e87f-4b75-877a-b085575bb528

**Review by Machi3mfl — CHANGES_REQUESTED** [review]
The `include time filters` feature doesn't work


https://github.com/user-attachments/assets/db487220-99ff-45c9-a0d7-619b0bf4eff7

**Review by Machi3mfl — APPROVED** [review]
https://github.com/user-attachments/assets/2e7db1e4-3574-428d-a5cb-497839dac89e

**Review by yenienserrano — APPROVED** [review]
LGTM 🟢 


https://github.com/user-attachments/assets/e0452081-807f-4cfe-bbf3-209d121ee3a4

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Fixed no-agent-alert spawn with selected agent in agent-welcome view [#7029](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7029)
 - Fixed security policy exception when it contained deprecated actions [#7042](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7042)
 - Fix export formatted csv data with special characters from tables [#7048](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7048)
+- Fixed the automatic page refresh [#7016](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7016)
```
Comment (Desvelao) [inline CHANGELOG.md:33]: **though**: Could you elaborate where the fix is affecting? It is not clear to me.

**Comment by guidomodarelli** [comment]
The resolution for this issue can be found in the pull request available at: https://github.com/wazuh/wazuh-dashboard-plugins/pull/7090.

This resolution also addressed a performance concern.

**Pull request closed without merging**